compiler/parser/src/parser.rs

```diff
@@ -189,12 +187,129 @@ pub fn parse_tokens(
     source_path: &str,
 ) -> Result<ast::Mod, ParseError> {
     let marker_token = (Default::default(), mode.to_marker(), Default::default());
-    let tokenizer = iter::once(Ok(marker_token))
+    let lexer = iter::once(Ok(marker_token))
         .chain(lxr)
         .filter_ok(|(_, tok, _)| !matches!(tok, Tok::Comment { .. } | Tok::NonLogicalNewline));
     python::TopParser::new()
-        .parse(tokenizer.into_iter())
-        .map_err(|e| crate::error::parse_error_from_lalrpop(e, source_path))
+        .parse(lexer.into_iter())
+        .map_err(|e| crate::parser::parse_error_from_lalrpop(e, source_path))
+}
+
+/// Represents represent errors that occur during parsing and are
+/// returned by the `parse_*` functions in the [parser] module.
+///
+/// [parser]: crate::parser
```

Comment:
this is private now so it should be removed, i.e:

```suggestion
/// Represents represent errors that occur during parsing and are
/// returned by the `parse_*` functions.
```

there's a couple of others similar to this, `cargo doc --no-deps -p rustpython-parser` should show them so you can remove them.